data/json/items/ranged.json

```diff
@@ -576,5 +576,32 @@
     "clip_size": 100,
     "reload": 0,
     "valid_mod_locations": [ [ "sling", 1 ] ]
+  },
+  {
+    "id": "fire_lance",
```

Comment:
This is utter nonsense.  
One, the difficulty to craft is ridiculously low.  
Two, the damage on this thing is crazy high.  
Three, the original is explicitly single-shot.  
The source weapon is a glorified firecracker on a stick, not a serious projectile weapon.

Leave it in MST and if you want a craftable early firearm look into casting a Gonne.
If you want a fire lance, look into making it some kind of single use distracting/stunning thing that deals minor damage.
You might say, "but that makes it useless", which is correct, they ARE useless for most individual purposes.

Reply:
I have no dog in this race. I nerfed it to oblivion.